lib/nviz/draw.c

```diff
@@ -230,13 +230,13 @@ int Nviz_draw_all(nv_data * data)
 	Nviz_draw_all_surf(data);
 
     if (draw_vect)
-	Nviz_draw_all_vect(data);
+	Nviz_draw_all_vect();
```

Comment:
The fn definition `int Nviz_draw_all_vect()` in L121 should be changed to `int Nviz_draw_all_vect(void)`

Reply:
updated

Reply:
What about the suggested change?

Reply:
Apparently resolved.